pipenv/cli/command.py

```diff
@@ -709,5 +709,24 @@ def verify(state):
     sys.exit(0)
 
 
+@cli.command(
+    short_help="Generate a requirements.txt from Pipfile.lock.",
+    context_settings=CONTEXT_SETTINGS,
+)
+@option("--dev", is_flag=True, default=False, help="Also add development requirements.")
+@pass_state
+def reqs(state, dev=False):
```

Comment:
Would you be open to calling this command `requirements` for clarity?

Reply:
Most definitely. I wanted to call it that, but requirements is a long word to type. Will update

Reply:
Thanks @ImreC for doing that, it will be more clear to non-English speakers I think why it exists.